operator/jsonnet/config.libsonnet

```
@@ -56,6 +56,13 @@ local utils = (import 'github.com/grafana/jsonnet-libs/mixin-utils/utils.libsonn
         ],
       },
 
+    local replaceType = function(type, replacement)
```

nit: add a small comment to the new function
```suggestion
        // replaceType updates the type of a dashboard. Used to 
        // transform "stat" graphs in "singlestat" to ensure OCP Console compatibility
        local replaceType = function(type, replacement)
```